scripts/changelog.ts

```diff
@@ -295,13 +295,15 @@ if (typeof config.to === 'undefined') {
       break;
     case 'product':
     case '@spartacus/product':
-    case '@spartacus/product/configurators':
-    case '@spartacus/product/configurators/common':
-    case '@spartacus/product/configurators/cpq':
-    case '@spartacus/product/configurators/variant':
-    case '@spartacus/product/configurators/textfield':
       config.library = '@spartacus/product';
       break;
+    case 'product-configurator':
+    case '@spartacus/product-configurator':
+    case '@spartacus/product-configurator/common':
```

Comment:
We don't need to add here all entry points. `product-configurator` + `@spartacus/product-configurator` is enough.

Reply:
ok, I see..Will remove